cpp/arcticdb/storage/s3/mock_s3_client.hpp

```diff
@@ -48,7 +48,7 @@ class MockS3Client : public S3ClientWrapper {
     //
     // The returned name looks like "{s3_object_name}#Failure_{operation_to_fail}_{error_to_fail_with}".
```

Comment:
This comment should be updated to include the `_{retryable}` clause you've added.